docs/csharp/local-functions-vs-lambdas.md

```diff
@@ -65,20 +65,20 @@ Third, the compiler can perform static analysis that enables local functions to
 definitely assign captured variables in the enclosing scope. Consider this example:
 
 ```csharp
-bool M()
+int M()
 {
     int y;
-    Local();
+    LocalFunction();
     return y;
 
-    void Local() => y = 0;
+    void LocalFunction() => y = 0;
 }
 ```
 
 The compiler can determine that `Local` definitely assigns `y` when called. Because `Local` is called before the `return` statement, `y` is definitiely
```

Comment:
As `Local` is renamed to `LocalFunction` in the example, please update the paragraph after the example as well.